port/unix/omrsysinfo.c

```diff
@@ -581,46 +584,66 @@ omrsysinfo_get_OS_type(struct OMRPortLibrary *portLibrary)
 #endif
 }
 
+#if defined (OSX)
+#define KERN_OSPRODUCTVERSION "kern.osproductversion"
+#endif
+
+#if defined(J9OS_I5)
+ /* 'V', 'R', 'M', '0', & null */
+#define RELEASE_OVERHEAD 5
+#define FORMAT_STRING "V%sR%sM0"
+#else
+/* "." and terminating null character */
+#define RELEASE_OVERHEAD 2
+#define FORMAT_STRING "%s.%s"
+#endif /* defined(J9OS_I5) */
 
 const char *
 omrsysinfo_get_OS_version(struct OMRPortLibrary *portLibrary)
 {
 	if (NULL == PPG_si_osVersion) {
 		int rc;
+#if !defined (OSX)
 		struct utsname sysinfo;
+#endif
 
 #ifdef J9ZOS390
 		rc = __osname(&sysinfo);
+#elif defined (OSX)
+			/* uname() on MacOS returns the version of the Darwin kernel.
```

Comment:
It looks like there is an extra tab for the comment. Could you align the comment with the code?  Thanks

Thanks for the comment as it is exactly what I was looking for!